hieradata/org/ncsa.yaml

```diff
@@ -404,84 +404,84 @@ unbound::backup_dns_servers:
+  - "[210.10.in-addr.arpa., 130.126.2.131]"
+  - "[211.10.in-addr.arpa., 130.126.2.131]"
+  - "[212.10.in-addr.arpa., 130.126.2.131]"
+  - "[213.10.in-addr.arpa., 130.126.2.131]"
+  - "[214.10.in-addr.arpa., 130.126.2.131]"
+  - "[215.10.in-addr.arpa., 130.126.2.131]"
+  - "[216.10.in-addr.arpa., 130.126.2.131]"
+  - "[217.10.in-addr.arpa., 130.126.2.131]"
+  - "[218.10.in-addr.arpa., 130.126.2.131]"
+  - "[219.10.in-addr.arpa., 130.126.2.131]"
+  - "[220.10.in-addr.arpa., 130.126.2.131]"
+  - "[221.10.in-addr.arpa., 130.126.2.131]"
+  - "[222.10.in-addr.arpa., 130.126.2.131]"
+  - "[223.10.in-addr.arpa., 130.126.2.131]"
+  - "[224.10.in-addr.arpa., 130.126.2.131]"
+  - "[225.10.in-addr.arpa., 130.126.2.131]"
+  - "[226.10.in-addr.arpa., 130.126.2.131]"
+  - "[227.10.in-addr.arpa., 130.126.2.131]"
+  - "[228.10.in-addr.arpa., 130.126.2.131]"
+  - "[229.10.in-addr.arpa., 130.126.2.131]"
+  - "[230.10.in-addr.arpa., 130.126.2.131]"
+  - "[231.10.in-addr.arpa., 130.126.2.131]"
+  - "[232.10.in-addr.arpa., 130.126.2.131]"
+  - "[233.10.in-addr.arpa., 130.126.2.131]"
+  - "[234.10.in-addr.arpa., 130.126.2.131]"
+  - "[235.10.in-addr.arpa., 130.126.2.131]"
+  - "[236.10.in-addr.arpa., 130.126.2.131]"
+  - "[237.10.in-addr.arpa., 130.126.2.131]"
+  - "[238.10.in-addr.arpa., 130.126.2.131]"
+  - "[239.10.in-addr.arpa., 130.126.2.131]"
+  - "[240.10.in-addr.arpa., 130.126.2.131]"
+  - "[241.10.in-addr.arpa., 130.126.2.131]"
+  - "[242.10.in-addr.arpa., 130.126.2.131]"
+  - "[243.10.in-addr.arpa., 130.126.2.131]"
+  - "[244.10.in-addr.arpa., 130.126.2.131]"
+  - "[245.10.in-addr.arpa., 130.126.2.131]"
+  - "[246.10.in-addr.arpa., 130.126.2.131]"
+  - "[247.10.in-addr.arpa., 130.126.2.131]"
+  - "[248.10.in-addr.arpa., 130.126.2.131]"
+  - "[249.10.in-addr.arpa., 130.126.2.131]"
+  - "[250.10.in-addr.arpa., 130.126.2.131]"
+  - "[251.10.in-addr.arpa., 130.126.2.131]"
+  - "[252.10.in-addr.arpa., 130.126.2.131]"
+  - "[253.10.in-addr.arpa., 130.126.2.131]"
+  - "[254.10.in-addr.arpa., 130.126.2.131]"
+  - "[255.10.in-addr.arpa., 130.126.2.131]"
+  - "[16.172.in-addr.arpa., 130.126.2.131]"
+  - "[17.172.in-addr.arpa., 130.126.2.131]"
+  - "[18.172.in-addr.arpa., 130.126.2.131]"
+  - "[19.172.in-addr.arpa., 130.126.2.131]"
+  - "[20.172.in-addr.arpa., 130.126.2.131]"
+  - "[21.172.in-addr.arpa., 130.126.2.131]"
+  - "[22.172.in-addr.arpa., 130.126.2.131]"
+  - "[23.172.in-addr.arpa., 130.126.2.131]"
+  - "[10.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[24.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[25.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[26.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[27.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[28.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[29.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[30.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
+  - "[31.172.in-addr.arpa., 141.142.2.2, 141.142.230.144]"
```

Comment:
Each line is an arrays of strings, not one single string.
correct format should be: `- ["22.172.in-addr.arpa.", "130.126.2.131"]`

Reply:
Ugh. That would have been bad...

Reply:
See also: https://github.com/lsst-it/puppet-unbound/blob/master/manifests/init.pp

Reply:
I did the conservative approach and simply disabled that lint rule for that section.

Reply:
Looks good! Thanks!